blog/index.html

```diff
@@ -88,6 +88,8 @@ <h5 class="card-header"><i class="fa fa-tags"></i> Tags</h5>
           <div class="carousel-item {% if forloop.first %}active{% endif %}">
             <img class="card-img-top mb-2" src="{{ s.cover-image }}" alt="spotlight image of {{ s.title }}">
             <div class="card-body">
+              <h4 class="card-title">{{ s.status }} Spotlight</h4>
+              <hr>
```

Comment:
In bootstrap, I don't think you can have multiple `card-title` headers. Will look into it, will likely need rearrange this a bit.